public/assets/css/form_products.css

```diff
@@ -0,0 +1,212 @@
+* {
+    margin: 0;
+    box-sizing: border-box;
+}
+
+form {

```

Comment:
please use classname instead of tags, otherwise all form tags will take this behavior